pkg/xray/resource_xray_license_policy.go

```diff
@@ -0,0 +1,168 @@
+package xray
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceXrayLicensePolicyV2() *schema.Resource {
+	return &schema.Resource{
+		SchemaVersion: 1,
+		Create:        resourceXrayPolicyCreate,
```

Comment:
Please use `CreateContext` , `ReadContext` etc. it's really easy. Those are the new APIs for this

Reply:
Fixed in https://github.com/jfrog/terraform-provider-xray/pull/5/commits/0f84780911c77818b2ccb10a5eb1d5999388664d